<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>703</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="CanRoute">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>通道1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>通道2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_62">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>定时器1:0x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_63">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>模式:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AccCode">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00000000</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Timing0">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>CAN通道:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>设备类型:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="EqupType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>USB_CAN</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>USB_CAN2</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="CanStart">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>启动</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_60">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>滤波方式:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FilterType">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>所有类型</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>标准帧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>扩展帧</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="WorkMode">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>正常模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>只听模式</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>自测模式</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>屏蔽码:0x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Timing1">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>14</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CanConnect">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_61">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>定时器0:0x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AccMask">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>FFFFFFFF</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_48">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>验收码:0x</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>691</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_65">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>索引号:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="EqupType_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="CanRestart">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>84</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>复位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_64">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>显示信息</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DataSendlineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>01 02 03 04 05 06 07 08</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_66">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>发送信息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendTimer">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>定时发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopSend">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>停止发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendOne">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>420</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>单次发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Clear">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>帧类型:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FreamType">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>标准帧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>扩展帧</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>帧格式:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="FreamFormat">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>数据帧</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>远程帧</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>360</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>帧ID:0x</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FreamID">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>00000080</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>703</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
